### PR TITLE
clean NewAppGwIngressController

### PR DESCRIPTION
**Pull request by dvrkps**

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -34,12 +34,12 @@ type AppGwIngressController struct {
 // NewAppGwIngressController constructs a controller object.
 func NewAppGwIngressController(kubeclient kubernetes.Interface, appGwClient network.ApplicationGatewaysClient, appGwIdentifier appgw.Identifier, k8sContext *k8scontext.Context) *AppGwIngressController {
 	controller := &AppGwIngressController{
+		appGwClient: appGwClient,
 		appGwIdentifier:  appGwIdentifier,
 		k8sContext:       k8sContext,
 		k8sUpdateChannel: k8sContext.UpdateChannel,
 	}
 	controller.eventQueue = NewEventQueue(controller.processEvent)
```
Comment (vramakrishnan) [inline pkg/controller/controller.go]: We could also move eventQueue initialization up.

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -34,12 +34,12 @@ type AppGwIngressController struct {
 // NewAppGwIngressController constructs a controller object.
 func NewAppGwIngressController(kubeclient kubernetes.Interface, appGwClient network.ApplicationGatewaysClient, appGwIdentifier appgw.Identifier, k8sContext *k8scontext.Context) *AppGwIngressController {
```
[the remark this answers is not in the document]
Comment (dvrkps) [reply pkg/controller/controller.go]: @asridharan @vramakrishnan Done.

File: pkg/controller/controller.go (hunk 1 above)
Comment (dvrkps) [reply pkg/controller/controller.go]: @vramakrishnan i think that is not true. https://play.golang.org/p/-VciWJJPDgz

Comment (vramakrishnan) [reply pkg/controller/controller.go]: You are right. Callback needs to be initialized in next step, the way it is today.

**Comment by asridharan** [comment]
@dvrkps `go vet` is failing with the following error:
```
https://travis-ci.org/Azure/application-gateway-kubernetes-ingress/builds/435772383#L952
```
You need remote the `Kubernetes` package from the import.

**Comment by dvrkps** [comment]
@asridharan Done.

**Pull request merged**